meta/main.yml

```diff
@@ -2,20 +2,20 @@
 galaxy_info:
   author: Nick Janetakis
   description: The official rvm role to install and manage your ruby versions.
-  company:
+  company: ''
   license: license (MIT)
   min_ansible_version: 2.2
 
   platforms:
-  - name: EL
-    versions:
-    - all
-  - name: Ubuntu
-    versions:
-    - all
+    - name: EL
```

Comment:
True, and I can remove it... it just makes for more legible YAML. But it's totally unrelated to the actual fix :) — would you like me to revert that change before merging?